docs/versioned-plugins/integrations/aws-v7.0.0.asciidoc

```diff
@@ -17,13 +17,13 @@ END - GENERATED VARIABLES, DO NOT EDIT!
 
 === AWS Integration Plugin {version}
 
-// include::{include_path}/plugin_header.asciidoc[]
```

Comment:
IMPORTANT: Note that the main page for an integration continues to the the `plugin_header.asciidoc` file.  Only the component plugins (inputs, outputs, filters, codecs) use the `plugin_header-integration.asciidoc` file.

Why? The integration itself is treated as a regular stand-alone plugin. We have to "manufacture" a distinction for the component plugins so that we can point up to the integration, point to the integration source repo, etc.